praw/objector.py

```diff
@@ -161,6 +162,22 @@ def objectify(
         if "json" in data and "errors" in data["json"]:
             errors = data["json"]["errors"]
             if len(errors) > 0:
+                for error in errors:
+                    if "ratelimit" in [
+                        str(index(error, 0)).lower(),
```

Comment:
Can we just check whether `error[0]` is `"RATELIMIT"`? So far every case where we want this functionality has `"RATELIMIT"` as the error type. If we find out later that there are other cases that don't fit this pattern, we can expand the check then.